.github/ISSUE_TEMPLATE/bug-template.yaml

```diff
@@ -0,0 +1,36 @@
+name: "\U0001F41B Bug Report"
+description: Report an issue or possible bug
+title: "\U0001F41B BUG:"
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Quick Bug Form
+        Thank you for taking the time to file a bug report! Please fill out this form as completely as possible.
+
+  - type: input
+    attributes:
+      label: What version of `Wrangler` are you using?
+      placeholder: 0.0.0
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: What operating system are you using?
+      placeholder: Mac, Windows, Linux
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Describe the Bug
+      description: A clear and concise description of the bug & steps to reproduce.
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Link to Minimal Reproducible Example (optional)
```

Comment:
I don't believe that a reproduction would be provided as a URL?
How about we remove the "steps to reproduce" from the `Describe the bug` section.
And change this to a `textarea` that is `Steps to reproduce`?

Reply:
There was talk previously about getting something like Stackblitz or Codesandbox templates working with Wrangler2, which would allow for repros/reprex. 

However, that can always be added later, I will make the changes for simplifying it as is.